packages/perseus/src/widgets/measurer.tsx

```diff
@@ -121,17 +123,19 @@ const Measurer: any = createReactClass({
         const graphieDiv = ReactDOM.findDOMNode(this.refs.graphieDiv);
         // @ts-expect-error - TS2769 - No overload matches this call. | TS2339 - Property 'empty' does not exist on type 'JQueryStatic'.
         $(graphieDiv).empty();
+        // @ts-expect-error - Argument of type 'Element | Text | null' is not assignable to parameter of type 'HTMLElement'.
         const graphie = (this.graphie = GraphUtils.createGraphie(graphieDiv));
 
-        const scale = [40, 40];
-        const range = [
+        const scale: Coord = [40, 40];
+        const range: [Interval, Interval] = [
```

Comment:
**idea**: I wonder if you can elide the types if you just add `as const` behind these literals?  I'm not sure which is better/more maintainable. 🤔

Reply:
The type names are useful as documentation. If you're just looking at a pair of numbers it's hard to know if `Coord` or `Interval` was intended.